polly/plugins/publish/validate_deadline_done.py

```diff
@@ -26,15 +26,21 @@ def process(self, instance):
             6: "Pending",
         }
 
-        url = api.Session["AVALON_DEADLINE"] + "/api/jobs?JobID=%s"
+        assert "AVALON_DEADLINE" in api.Session, ("Environment variable "
+                                                  "missing: 'AVALON_DEADLINE'")
+        avalon_deadline = api.Session["AVALON_DEADLINE"]
```

Comment:
I thought we said to make it `AVALON_DEADLINE`? Uppercase reflects that it is both static, but also an environment variable in this case.

Also it'd be ideal to keep the submit file within the same convention.